code/game/turfs/simulated/floor/light_floor.dm

```diff
@@ -25,8 +26,20 @@
 
 /turf/simulated/floor/light/BeforeChange()
 	set_light(0)
+	STOP_PROCESSING(SSobj, src)
 	..()
 
+/turf/simulated/floor/light/process()
+	if(power_check() || !on)
+		return
+	extinguish_light()
```

Comment:
We have a signal for area power changing or similar, look what light switches and stuff use.

*Please* dont make turfs process.

Reply:
done